applications/jonny5/src/j5_balcraw_worker.erl

```diff
@@ -0,0 +1,118 @@
+%%%-------------------------------------------------------------------
+%%% File    : j5_balacne_crawler.erl
+%%% Description : Jonny5 module for disconnect calls when account
+%%% balance drops below zero
+%%%-------------------------------------------------------------------
+-module(j5_balcraw_worker).
+
+-export([start/0]).
+
+-export([maybe_disconnect_account/1]).
+-export([disconnect_account/1]).
+
+-include("jonny5.hrl").
+
+-define(SERVER, ?MODULE).
+-define(DEFAULT_INTERACCOUNT_DELAY, 10).
+
+-spec start() -> no_return().
+start() ->
+    case j5_channels:accounts() of
+        [] -> exit("no accounts");
```

Comment:
use atoms for enumerable returns: strings are near impossible to check statically (`make dialyze`).